src/Fixer/FunctionNotation/PhpdocToParamTypeFixer.php

```diff
@@ -38,7 +38,7 @@ final class PhpdocToParamTypeFixer extends AbstractFixer implements Configuratio
     const MINIMUM_PHP_VERSION = 70000;
 
     /**
-     * @var array
+     * @var array{int, string}[]
```

Comment:
Shouldn't it be `array<array<int, string>>`?

Reply:
no
`array<int, string>` means int as index, string as value
`array{int, string}` means a tuble of 2 elements, int and string

Reply:
Ah yes I read too fast.

Is this syntax defined in some standard, or well understood by static analysis tools at least?

Reply:
i took it from phpstan

Reply:
TIL, thanks :)